include/net/ilnp6.h

```diff
@@ -147,8 +147,8 @@ struct ilcc_table {
 /*Structure for LU message*/
 struct lu_data {
       uint64_t		l64;
-      uint32_t		prec;
-      uint32_t		lifetime;
+      uint16_t		prec;
```

Comment:
Preference type is okay to me but for lifetime type please see my next comment. 
According to RFC 6743, the preference numbers in the ICMPv6 messages are identical to preference values for L64s both in syntax and semantics. But, it would be great that this is tested first (although I think we are not syntactically overlapping any registers against these types), as LU update is quite pivotal to ILNP operation in mobility and multihoming.